ui/views.py

```diff
@@ -70,8 +71,10 @@ def default_js_settings(request):
         ),
         "support_email_address": settings.EMAIL_SUPPORT,
         "ga_dimension_camera": settings.GA_DIMENSION_CAMERA,
+        "video_annotations": video_annotations_feature_flag,
```

Comment:
Do we need this redundancy? Can we just check for `FEATURES["VIDEOJS_ANNOTATIONS"]` in the template?